src/mapgen.cpp

```diff
@@ -386,6 +386,8 @@ MgStoneType MapgenBasic::generateBiomes()
 	u32 index = 0;
 	MgStoneType stone_type = MGSTONE_STONE;
 
+	noise_filler_depth->perlinMap2D(node_min.X, node_min.Z);
```

Comment:
I feel 'filler depth' should now be a biomesparam (if this is still meaningful in your new proposal) seems the correct place for it instead of mapgen base terrain parameters.